lib/jsdom/events/EventException.js

```diff
@@ -0,0 +1,27 @@
+﻿"use strict";
```

Comment:
This file needs a TODO that this whole concept should probably be killed, as doesn't exist in the spec.